### PR TITLE
Fixes #17177: Distribute node-to-relay script with the plugin 

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/17177

### CODE REVIEW

**Comment by ElaadF** [comment]
Commit modified

File: scale-out-relay/src/main/resources/server-node-to-relay [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
```
Comment (peckpeck) [inline scale-out-relay/src/main/resources/server-node-to-relay:1]: don't hesitate to use /bin/bash here, its better

File: scale-out-relay/src/main/resources/server-node-to-relay [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+#####################################################################################
```
Comment (peckpeck) [inline scale-out-relay/src/main/resources/server-node-to-relay:3]: Please replace this with SPDX heades

File: scale-out-relay/src/main/resources/server-node-to-relay [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+#
+# Rudder is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
```
Comment (peckpeck) [inline scale-out-relay/src/main/resources/server-node-to-relay:35]: Please add rudder command docs, example from "rudder agent check" : 
```
# @description check if rudder agent has no problem and is running properly
# @man Check that rudder agent is working properly.
# @man - generate missing UUID or keys
# @man - kill cfengine if there are too many processes
# @man - run cfengine if its daemon is missing
# @man - clean lock file if it is too big
# @man - check that promises have been properly copied
# @man - will sleep a random time (max half the run interval) when not run in interactive mode
# @man +
# @man *Options*:
# @man +
# @man *-q*: run the agent in quiet mode (display only error and warning messages)
# @man +
# @man *-c*: run the agent without color output

```

**Comment by ElaadF** [comment]
Commit modified

File: scale-out-relay/Makefile [hunk 4]
```diff
@@ -3,8 +3,10 @@
...
 #
 
-FILES =
+FILES = $(NAME)/server-node-to-relay
 SCRIPTS = postinst
 
 include ../makefiles/common-scala-plugin.mk
 
+target/$(NAME)/server-node-to-relay:
+	cp ./src/main/resources/server-node-to-relay target/$(NAME)/
```
Comment (peckpeck) [inline scale-out-relay/Makefile:12]: Missing final end of line.
Particularly important for old tools like make and the shell

**Comment by peckpeck** [comment]
looks good to me

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**